srcpkgs/GammaRay/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'GammaRay'
+pkgname=GammaRay
+version=2.11.1
+revision=1
+wrksrc="gammaray-${version}"
+build_style=cmake
+configure_args="-DGAMMARAY_INSTALL_QT_LAYOUT=true -DECM_MKSPECS_INSTALL_DIR=/usr/lib/qt5/mkspecs/modules -DPLUGIN_INSTALL_DIR=/usr/lib/qt5/plugins/gammaray"
+hostmakedepends="qt5-host-tools qt5-devel"
+makedepends="qt5-devel"
+short_desc="Tool to poke around in a Qt-application"
+maintainer="toluschr <toluschr@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/KDAB/GammaRay"
+distfiles="${homepage}/releases/download/v${version}/gammaray-${version}.tar.gz"
+checksum=87a1d72ad1ad6d1a0156c54a85b0976ab38c6a64136458ca7c4ee491566d25d0
+
+LDFLAGS+=" -Wl,-no-fatal-warnings"
```

Comment:
The first occurence of `LDFLAGS` doesn't need `+=`
```sh
LDFLAGS="-Wl,-no-fatal-warnings"
```

Reply:
I thought it added to the environment's FLAGS, sorry!

Reply:
The template will be sourced first. The various hooks and setup will prepend/append necessary arguments into various flags, including but not limiting to `configure_args` `LDFLAGS`

Reply:
That makes sense, thanks.